### ISSUE TITLE
Update `fxa.metrics_uid_secret_key` to match the HMAC key used for Amplitude metrics

### ISSUE THREAD

**Issue by philbooth** [mozilla-services/tokenserver#112]
Spinning out from mozilla/fxa-amplitude-send#24 as a separate issue because there's a whole load of discussion there that isn't relevant here.

This line in the tokenserver references a HMAC key that we want to change:

https://github.com/mozilla-services/tokenserver/blob/master/tokenserver/views.py#L117

We want it to be the same as the key used for the Amplitude metrics here:

https://github.com/mozilla/fxa-amplitude-send/blob/master/amplitude.py#L15

Before making the change we're testing it in stage, hence the separate email thread about the PI request. That testing needs to happen in three parts:

1. Softvision connect devices to stage servers.
2. We update the tokenserver config so that a new HMAC key is used.
3. Softvision carry on testing against stage to verify that nothing bad happened in any of the clients.

Currently step 1 of the above has happened, so we want to do step 2 then let them know to proceed with step 3.

After step 3 has happened and we have a green light from Softvision, we can move on to making the change in prod.

**Comment by jrgm** [comment]
Stage tokenserver config has been updated to use the same hmac key as fxa-amplitude-send. Let me know if there's any problem with that, and I can debug, or switch back.

**Comment by philbooth** [comment]
Mihai has now tested this and reports a green light:

> We also focused our exploratory and regression testing on multiple account related actions: Sync, account management changes, and data manipulation and syncing in order to cover the PI request to test the "Update `fxa.metrics_uid_secret_key` to match the HMAC key used for Amplitude metrics #112 ". During testing we found no issues that may be trigered by this server change and we suggest a GREEN status on it. 
>
> In order to test we followed the mentioned steps in the ticket's description after we got the information from John (via train tracking bug comment) that the changes were pushed. The devices used for testing it were as following:
>
> * Windows 8 - 64 Bit
> * Samsung Galaxy S7 - Android 7.0
> * Samsung Galaxy S6 EDGE - Android 7.0
> * Xiaomi  Redmi  Note 3 -  Android 6.0.1
> * Samsung Galaxy Tab Active - Android 5.1.1
> * Nexus 6P - Android 8.0
> * Google Pixel - Android 8.0
> * HTC 10 - Android 7.0
> * iPad Mini4 - Fw 11.3 (15E5167f) 

@jrgm, before we go ahead and make the change in prod I want to check with any interested parties whether they want us to hold off for any metrics-related reasons, maybe they're part way through some crucial analysis of Sync telemetry data or whatever.

Afaik, "interested parties" === @irrationalagent, @mhammond and @thomcc in this case (there's also an old email thread I need to dig up and respond to).

**Comment by irrationalagent** [comment]
I'm good on this. have stuff that I'm planning but not in flight yet.

**Comment by philbooth** [comment]
@jrgm, as per the back-end meeting last night, let's aim for Wednesday next week to make this change in prod. I've separately notified the Sync devs that's the plan via email, with the caveat that it depends on your bandwidth so there's leeway for slippage if required.

**Comment by philbooth** [comment]
Postponed until Monday, because AWS outage.

**Comment by philbooth** [comment]
This change has now been made in prod and my `Weave.Service.identity.hashedUID()` in the browser correctly matches my `user_id` in Amplitude. Closing this.

**Comment by philbooth** [comment]
Just to wrap it all up, the fruits of this labour can now be seen in Amplitude:

<img width="1123" alt="Screenshot of a chart in Amplitude showing events from Sync telemetry data" src="https://user-images.githubusercontent.com/64367/36245745-751df6b0-1224-11e8-934f-67c34b680704.png" />

https://analytics.amplitude.com/org/31251/chart/o7js9en
